packages/progress-indicator/test/lion-progress-indicator.test.js

```diff
@@ -1,23 +1,155 @@
+      expect(el.getAttribute('aria-label')).to.equal('foo');
+      el.setAttribute('aria-label', 'bar');
+      expect(el.getAttribute('aria-label')).to.equal('bar');
+      el.removeAttribute('aria-label');
+      await el.updateComplete;
+      expect(el.getAttribute('aria-label')).to.equal('Loading');
+    });
+
+    it('can override a label with "aria-labelledby"', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator aria-labelledby="foo-id"></lion-progress-indicator> `,
+      );
+      await el.localizeNamespacesLoaded;
+      expect(el.getAttribute('aria-labelledby')).to.equal('foo-id');
+      // expect(el.hasAttribute('aria-label')).to.be.false;
+      el.setAttribute('aria-labelledby', 'bar-id');
+      expect(el.getAttribute('aria-labelledby')).to.equal('bar-id');
+      expect(el.hasAttribute('aria-label')).to.be.false;
+      el.removeAttribute('aria-labelledby');
+      await el.updateComplete;
+      expect(el.hasAttribute('aria-labelledby')).to.be.false;
+      expect(el.getAttribute('aria-label')).to.equal('Loading');
+    });
+  });
+
+  describe('determinate', async () => {
+    it('is determinate when has value attribute', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="25" aria-label="foo"></lion-progress-indicator> `,
+      );
+      expect(el.indeterminate).to.be.false;
+    });
+
+    it('can update value', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="25" aria-label="foo"></lion-progress-indicator> `,
+      );
+      expect(el.getAttribute('aria-valuenow')).to.equal('25');
+      el.value = 30;
+      await el.updateComplete;
+      expect(el.getAttribute('aria-valuenow')).to.equal('30');
+    });
+
+    it('can update min', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="50" aria-label="foo"></lion-progress-indicator> `,
+      );
+      expect(el.getAttribute('aria-valuemin')).to.equal('0');
+      el.min = 30;
+      await el.updateComplete;
+      expect(el.getAttribute('aria-valuemin')).to.equal('30');
+    });
+
+    it('can update max', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="50" aria-label="foo"></lion-progress-indicator> `,
+      );
+      expect(el.getAttribute('aria-valuemax')).to.equal('100');
+      el.max = 70;
+      await el.updateComplete;
+      expect(el.getAttribute('aria-valuemax')).to.equal('70');
+    });
+
+    it('min & max limits value', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="150" aria-label="foo"></lion-progress-indicator> `,
+      );
+      // sets to default max: 100
+      expect(el.getAttribute('aria-valuenow')).to.equal('100');
+      el.value = -20;
+      await el.updateComplete;
+      // sets to default min: 0
+      expect(el.getAttribute('aria-valuenow')).to.equal('0');
+    });
+  });
+
```

Comment:
I think we only need a test with:
- 
- a TODO for aria-valuetext => allow a function like `_computeAriaValueText({ value:number; min:number; max:number;}): string;`